src/main/resources/default/assets/scripts/list-layout-switch.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
```

Comment:
Remove copyright declaration as it will be rendered and this file is not used independently but as part of another js file.